gluon/html.py

```diff
@@ -437,9 +429,20 @@ def verifyURL(request, hmac_key=None, hash_vars=True, salt=None, user_signature=
         >>> verifyURL(r, 'key')
         False
 
+    Args:
+        user_signature:
```

Comment:
I just append args automatically with pycharm sorry if they are not in the
proper order... Can be fixed latter...

On Tue, Mar 15, 2016 at 4:35 PM, niphlod <notifications@github.com> wrote:

> In gluon/html.py
> <https://github.com/web2py/web2py/pull/1219#discussion_r56235807>:
>
> > @@ -437,9 +429,20 @@ def verifyURL(request, hmac_key=None, hash_vars=True, salt=None, user_signature=
> >          >>> verifyURL(r, 'key')
> >          False
> >
> > +    Args:
> > +        user_signature:
>
> args in wrong order, too
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/web2py/web2py/pull/1219/files/00632a9dec8ce371ea42b0a7ae2bdbe2d94aea4c#r56235807>
>